node/service/src/chain_spec.rs

```diff
@@ -112,6 +112,11 @@ pub fn rococo_config() -> Result<PolkadotChainSpec, String> {
 	PolkadotChainSpec::from_json_bytes(&include_bytes!("../res/rococo.json")[..])
 }
 
+/// This is a temporary testnet that uses the same runtime as rococo.
+pub fn wococo_config() -> Result<PolkadotChainSpec, String> {
+	PolkadotChainSpec::from_json_bytes(&include_bytes!("../res/wococo.json")[..])
```

Comment:
What exactly did you change to have different genesis hash?

Reply:
It is built from a later version of substrate which by itself should generate a different WASM blob, and that get includes into storage as part of genesis. But if I diff `rococo.json` and `wococo.json` there are more storage entries that have different values.

```
❯ ./target/release/polkadot --chain wococo --tmp 
[...]
2021-04-21 21:34:57  💾 Database: RocksDb at /tmp/substrategSXl1c/chains/wococo/db
2021-04-21 21:34:57  ⛓  Native runtime: rococo-231 (parity-rococo-v1.5-0.tx0.au0)
2021-04-21 21:34:57  🔨 Initializing Genesis block/state (state: 0x3ea4…a8ba, header-hash: 0x89f5…694e)
```

```
❯ ./target/release/polkadot --chain rococo --tmp
[...]
2021-04-21 21:35:10  💾 Database: RocksDb at /tmp/substrateavbCDJ/chains/rococo_v1_5/db
2021-04-21 21:35:10  ⛓  Native runtime: rococo-231 (parity-rococo-v1.5-0.tx0.au0)
2021-04-21 21:35:10  🔨 Initializing Genesis block/state (state: 0x63b1…064d, header-hash: 0x1611…a215)
```

Additionally they also have a different `protocolId` in the chain spec, which gets used to separate the networks (and also a different token name but that's just metadata).